docs/configure_your_operations.md

```diff
@@ -46,22 +49,24 @@ class Book implements ResourceInterface
 
 It will configure this route for your `index` operation.
 
-| Name                  | Method          | Path    |
-|-----------------------|-----------------|---------|
-| app_book_index        | GET             | /books  |
+| Name           | Method | Path   |
+|----------------|--------|--------|
+| app_book_index | GET    | /books |
 
 On your Twig template, these variables are available
 
-| Name      | Type                                     |
-|-----------|------------------------------------------|
-| resources | Pagerfanta\Pagerfanta                    |
-| books     | Pagerfanta\Pagerfanta                    |
-| operation | Sylius\Component\Resource\Metadata\Index |
-| app       | Symfony\Bridge\Twig\AppVariable          |
+| Name              | Type                                      |
+|-------------------|-------------------------------------------|
+| resources         | Pagerfanta\Pagerfanta                     |
+| books             | Pagerfanta\Pagerfanta                     |
+| operation         | Sylius\Resource\Metadata\Index            |
+| resource_metadata | Sylius\Resource\Metadata\ResourceMetadata |
+| app               | Symfony\Bridge\Twig\AppVariable           |
 
 ### Use a grid for your index operation
 
-To use a grid for you operation, you need to install the [Sylius grid package](https://github.com/Sylius/SyliusGridBundle/)
+To use a grid for you operation, you need to install
```

Comment:
Is line spit just for raw readability? It's rendered as single line.

![image](https://github.com/Sylius/SyliusResourceBundle/assets/870747/2b3a59a7-5633-44d7-ad78-8872f381c02d)

Reply:
Yeah, PHPStorm did that when I use a reformat for a table.